enrichment-web/src/main/java/eu/europeana/enrichment/web/controller/AdministrationController.java

```diff
@@ -93,14 +93,17 @@ public ResponseEntity<String> updateStories(
 	@RequestMapping(value = "/administration/updateStoriesFromTranscribathon", method = {RequestMethod.POST},
 			consumes = {MediaType.APPLICATION_JSON_VALUE}, produces = MediaType.TEXT_PLAIN_VALUE)
 	public ResponseEntity<String> updateStoriesFromTranscribathon(
-			@RequestBody String storiesIds,
+			@RequestBody List<String> storyIdsList,
 			HttpServletRequest request) throws Exception {
 		
 		verifyWriteAccess(Operations.CREATE, request);
-		List<String> storyIdsList = new ArrayList<String>(Arrays.asList(HelperFunctions.toArray(storiesIds,",")));
+		List<String> fieldsToUpdate = new ArrayList<>();

```

Comment:
These are the fields for the story, which are additionally checked in the update, because if those fields are changed, some other entities need to be changed/deleted. E.g. if the transcription of the story has changed, the annotations related to the old transcription need to be removed. For the items, the only field we currently compute named entities is transcription, so no need to define the fields for it.